hr_emergency_contact/models/hr_employee.py

```diff
@@ -3,20 +3,17 @@
 # © 2016 OpenSynergy Indonesia
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields
+from odoo import models, fields
 
 
 class HrEmployee(models.Model):
 
     _inherit = 'hr.employee'
 
     emergency_contact_ids = fields.Many2many(
-        string='Emergency Contacts',
         comodel_name='res.partner',
-        relation='rel_employee_emergency_contact',
```

Comment:
@leemannd if you remove this parameter is it possible there will be an error if user migrate from previous version?